bridge/pom.xml

```diff
@@ -85,6 +85,22 @@
 			<artifactId>rest-assured</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- QwandaQ already has this but native build will fail so adding these here too-->
```

Comment:
Have another look at this. See if you can find a way to remove it in all the child pom files.

Reply:
fixed